apps/webapp/tailwind.config.js

```diff
@@ -87,6 +87,14 @@ module.exports = {
             fontWeight: "500",
           },
         ],
+        "2sm": [
```

Comment:
@samejr I can't tell from the name of this how big it is in relation to the other sizes… naming is hard, but I think we can do better than this.